app/serializers/location_free_spaces_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class LocationFreeSpacesSerializer
+  include JSONAPI::Serializer
+
+  set_type :locations
+
+  attributes :title
+
+  meta do |object, params|
+    {
+      populations: params[:populations][object.id],
```

Comment:
```suggestion
      capacity_over_days: params[:capacity_over_days][object.id],
```

Reply:
The contract for this has already been agreed with the front end devs.